src/main/kotlin/thatline/localup/storage/service/StorageService.kt

```
@@ -0,0 +1,150 @@
+        // fileId와 userId로 파일 찾기
+        val fileEntity = fileRepository.findByIdAndUserIdAndDeletedFalse(fileId, userId)
+            ?: return null
+        
+        try {
+            val path = Paths.get(fileEntity.filePath)
+            val resource = UrlResource(path.toUri())
+            
+            if (resource.exists() && resource.isReadable) {
+                return Pair(resource, fileEntity.fileName)
+            }
+        } catch (ex: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)